eval.cpp

```diff
@@ -784,6 +793,11 @@ namespace Sass {
       Expression* ex = operator()(var);
       return evacuate_quotes(interpolation(ex));
 
+    } else if (Parent_Selector* var = dynamic_cast<Parent_Selector*>(s)) {
+
+      Expression* ex = operator()(var);
```

Comment:
actually i looked at the code again.  yeah, i think they are the same

      Expression* ex = var->perform(this);